VideoApi/VideoApi/Controllers/ConferenceManagementController.cs

```diff
@@ -190,7 +190,7 @@ public async Task<IActionResult> TransferParticipantAsync(Guid conferenceId, Tra
 
         private string TransferFromRoomType(ParticipantBase participant)
         {
-            return (participant.UserRole == UserRole.QuickLinkParticipant || participant.UserRole == UserRole.QuickLinkObserver) && participant.CurrentConsultationRoom != null ? participant.CurrentConsultationRoom.Label : RoomType.WaitingRoom.ToString();
+            return (participant.UserRole == UserRole.QuickLinkParticipant || participant.UserRole == UserRole.QuickLinkObserver) && participant.CurrentConsultationRoom != null && !string.IsNullOrWhiteSpace(participant.CurrentConsultationRoom.Label) ? participant.CurrentConsultationRoom.Label : RoomType.WaitingRoom.ToString();
```

Comment:
```suggestion
            return (participant.UserRole == UserRole.QuickLinkParticipant || participant.UserRole == UserRole.QuickLinkObserver) && !string.IsNullOrWhiteSpace(participant.CurrentConsultationRoom?.Label) ? participant.CurrentConsultationRoom.Label : RoomType.WaitingRoom.ToString();
```

Reply:
Would this work?

Reply:
I can try this, but I felt the other way to write down this logic was a bit more readable

Reply:
all the unit test passed, one of the test was checking when the currentconsutationroom property is null so I guess it should work